src/components/pages/Budget/BudgetForm.js

```diff
@@ -167,14 +167,18 @@ export const BudgetForm = ({ budget, handleCloseModal, create }) => {
               <Button variant='primary' type='submit' disabled={isSubmitting}>
                 {isSubmitting ? 'Saving...' : 'Save Changes'}
               </Button>
-              <Button
-                variant='danger'
-                disabled={isSubmitting}
-                onClick={handleDeleteBudget}
-                type='submit'
-              >
-                {isSubmitting ? 'Deleting...' : 'Delete'}
-              </Button>
+              {!create ? (
+                <Button
+                  variant='danger'
+                  disabled={isSubmitting}
+                  onClick={handleDeleteBudget}
+                  type='submit'
+                >
+                  {isSubmitting ? 'Deleting...' : 'Delete'}
+                </Button>
+              ) : (
+                ''
+              )}
```

Comment:
I have already changed this to this
```suggestion
             {!create && (
                <Button
                  variant='danger'
                  disabled={isSubmitting}
                  onClick={handleDeleteBudget}
                >
                  {isSubmitting ? 'Deleting...' : 'Delete'}
                </Button>
              )}
```